changelogs/fragments/a_test_rmv_alpine_317.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - ansible-test, remove Alpine 3.17 from remotes/CI
```

We don't include CI changes in the changelog, since they don't make it into the sdist, just the ansible-test changes:

```suggestion
  - ansible-test - remove Alpine 3.17 from remotes
```